Backend/Test_cases/test_cases.py

```diff
@@ -0,0 +1,84 @@
+from fastapi.testclient import TestClient
+from main import app
+
+client = TestClient(app)
+
+data = {
+    "ngoName": "string",
+    "contactPersonName": "string",
+    "contactPersonEmail": "user@example.com",
+    "contactPersonPhone": "7578061886",
+    "contactPersonPassword": "string",
+    "location": "string",
+    "pincode": "784115"
+}
+
+
+def test_create_ngo():
+    response = client.post("/create_ngo", json=data)
+    assert response.status_code == 200
+
+
+def test_create_ngo_with_wrong_phone_number():
+    response = client.post("/create_ngo", json={
+        "ngoName": "string",
+        "contactPersonName": "string",
+        "contactPersonEmail": "user@example.com",
+        "contactPersonPhone": "757806188",
+        "contactPersonPassword": "string",
+        "location": "string",
+        "pincode": "784115"
+    })
+    assert response.status_code == 200
```

Comment:
It can't be 200 here. it could be 422 unprocessiable entity or 500 internal server error may be not 200